kapitan/targets.py

```diff
@@ -563,8 +573,11 @@ def schema_validate_compiled(targets, compiled_path, inventory_path, schema_cach
     worker = partial(schema_validate_kubernetes_output, cache_dir=schema_cache_path)
     pool = multiprocessing.Pool(parallel)
 
+    if targets_path == None:
+        targets_path = inventory_path
```

Comment:
Shouldn't this be set to `os.path.join(inventory_path, "targets")` ? 
Or am I missing something?

Reply:
Yeah I missed out here, will fix it.